autopr/tests/test_validators.py

```diff
@@ -495,22 +495,6 @@ def repo_to_codebase(
 @@ -67,1 +69,23 @@
          return filenames_and_contents
 +
-+    def _repo_to_files_and_token_lengths(
-+        self,
-+        repo_tree: git.Repo,
-+        excluded_files: list[str] = None,
-+    ) -> list[tuple[str, int]]:
-+        files_with_token_lengths = []
-+        for blob in repo_tree.traverse():
-+            if blob.type == 'tree':
-+                continue
-+            if excluded_files is not None and blob.path in excluded_files:
-+                continue
-+            content = blob.data_stream.read().decode()
-+            token_length = len(self.rail_service.tokenizer.encode(content))
-+            files_with_token_lengths.append((blob.path, token_length))
-+        return files_with_token_lengths
-+
```

Comment:
You don't need to remove it from the test cases.
The reason you had to merge main afterwards is because I had just merged a huge tests refactor yesterday. As it stands this PR is +395 -137, which means something's wrong.

Drop the merge main commit, and rebase on the main branch, editing the changes to `test_validators.py` out of the commit. Alternatively, enable ["Allow edits from maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork), and I can do it.

Reply:
I dropped the last merge and branch skip-non-utf8 is up to date with the main. I also enabled "Allow edits by maintainers", in case there are any issues.